app/questionnaire/questionnaire_schema.py

```diff
@@ -893,3 +893,12 @@ def _get_rules_section_dependencies(
                 )
 
         return rules_section_dependencies
+
+    def _get_related_answers_for_section(self, section_id: str) -> Optional[list[str]]:
+        if summary := self.get_section(section_id).get("summary"):  # type: ignore
```

Comment:
You can reuse `get_summary_for_section` method here.

Reply:
This was easy enough to fix quickly so I pushed the changes now.